files/usr/share/cinnamon/applets/sound@cinnamon.org/applet.js

```diff
@@ -965,6 +966,9 @@ class CinnamonSoundApplet extends Applet.TextIconApplet {
         this._volumeMax = this._sound_settings.get_int(MAXIMUM_VOLUME_KEY) / 100 * this._control.get_vol_max_norm();
         this._volumeNorm = this._control.get_vol_max_norm();
 
+        this._mediakeysStep_settings = new Gio.Settings({ schema_id: VOLUME_STEP_DIR });
```

Comment:
There is already a settings object for this schema - `this._sound_settings` - use it instead